src/main/scala/uk/gov/hmrc/message2/library/model/MessageRESTFormats.scala

```diff
@@ -85,6 +85,7 @@ object MessageRESTFormats extends RestFormats with BSONObjectIdFormats with Aler
         val rendererService = externalRef.source.toUpperCase match {
           case "2WSM" => "two-way-message"
           case _      => "external-message-adapter"
```

Comment:
Not ideal when reusing that part in the quadient module (should be `quadient` instead of `external-message-adapter`) - but that part is unused so it shouldn't be too bad